resources/linux/create_desktop_file.sh

```diff
@@ -14,5 +14,6 @@ Terminal=false
 Type=Application
 Icon=${FULL_PATH}/icon.svg
 Categories=Network;InstantMessaging;
+MimeType=x-scheme-handler/mattermost;
```

Comment:
people can modify the scheme in [this file](https://github.com/mattermost/desktop/blob/master/electron-builder.json#L24), this script should read from there by using `jq`